engine/src/main/scala/cromwell/engine/workflow/WorkflowDockerLookupActor.scala

```diff
@@ -104,10 +105,18 @@ class WorkflowDockerLookupActor private[workflow](workflowId: WorkflowId,
 
   whenUnhandled {
     case Event(DockerHashActorTimeout(request), data) =>
-      val reason = new Exception(s"Timeout looking up docker hash")
-      data.hashRequests(request) foreach { _ ! WorkflowDockerLookupFailure(reason, request) }
-      val updatedData = data.copy(hashRequests = data.hashRequests - request)
-      stay() using updatedData
+      val reason = new Exception(s"Timeout looking up hash for Docker image ${request.dockerImageID}")
+      data.hashRequests.get(request.dockerImageID) match {
+        case Some(requestsAndReplyTos) =>
+          requestsAndReplyTos foreach { case RequestAndReplyTo(_, replyTo) =>
+            replyTo ! WorkflowDockerLookupFailure(reason, request)
+          }
+          val updatedData = data.copy(hashRequests = data.hashRequests - request.dockerImageID)
+          stay() using updatedData
+        case None =>
+          val message = s"Unable to find requesters for timed out lookup of Docker image '${request.dockerImageID}'"
```

Comment:
Did we also want to print out the `request -> waiting actors` mapping here? Since we're hoping not to see this again, plenty of verbosity would hopefully be helpful rather than just noisy?

Reply:
discussed with faces